models/transaction.js

```diff
@@ -60,10 +60,20 @@ export default class Transaction extends BaseModel {
     async countConfirmed(type) {
         try {
             const res = await this.get(`SELECT type, COUNT(*) FROM ${this.tableName} WHERE type = ? AND status = 'confirmed' GROUP BY type`, [type]);
-            return res || 0;
+            return res ? res['COUNT(*)'] : 0;
```

Comment:
does this actually work? I think it should just use `SELECT COUNT(*) AS ct FROM ...` in the query, and then `return res ? res.ct : 0;`